### PR TITLE
[SMP-1201] Add air-gapped installation topic

### PR DESCRIPTION
**Pull request by brian-f-harness**
Add SMP air-gapped installation topic

For reviewers, [a preview](https://6446da127a79db00967fa5ce--harness-developer.netlify.app/docs/self-managed-enterprise-edition/self-managed-helm-based-install/install-in-an-air-gapped-environment) is available.

## What Type of PR is This?

- [ ] Issue
- [x] Feature
- [ ] Maintenance/Chore

If tied to an Issue, list the Issue(s) here:

* *Issue(s)*

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [x] Tested Locally
- [ ] *Optional* Screen Shoot.

### CODE REVIEW

**Review by pratmit — CHANGES_REQUESTED** [review]
Modified the file to remove the cluster requirements.

**Review by swapnil-harness — COMMENTED** [review]
Please hold off on this for one more day, there will be some delta, I'll confirm by tomorrow.

**Comment by brian-f-harness** [comment]
> Please hold off on this for one more day, there will be some delta, I'll confirm by tomorrow.

@swapnil-harness is there an update on this feature?

**Comment by swapnil-harness** [comment]
> > Please hold off on this for one more day, there will be some delta, I'll confirm by tomorrow.
> 
> @swapnil-harness is there an update on this feature?

Hi Brian, yes made the changes in code and updated the doc as well. Thanks!

**Review by pratmit — APPROVED** [review]
LGTM

File: docs/self-managed-enterprise-edition/self-managed-helm-based-install/install-in-an-air-gapped-environment.md [hunk 1]
```diff
@@ -0,0 +1,99 @@
+---
+title: Install in an air-gapped environment
+description: Learn how to install the Harness Self-Managed Enterprise Edition using Helm in an air-gapped environment. 
+# sidebar_position: 6
+---
+
+This document explains how to use Helm to install the Harness Self-Managed Enterprise Edition in an air-gapped environment and how to obtain and transfer Docker images to a private registry secure access. The steps include pulling Docker images, saving images as .tgz files, uploading to Google Cloud storage, downloading Helm charts, and pushing charts to your private repositories. This process ensures secure and seamless deployment of the Harness Self-Managed Enterprise Edition in restricted, offline environments.
```
Comment (douglas-j-bothwell) [inline docs/self-managed-enterprise-edition/self-managed-helm-based-install/install-in-an-air-gapped-environment.md:7]:
```suggestion
This topic explains how to use Helm to install the Harness Self-Managed Enterprise Edition in an air-gapped environment and how to obtain and transfer Docker images to a private registry with secure access. The steps include pulling Docker images, saving images as .tgz files, uploading to Google Cloud storage, downloading Helm charts, and pushing charts to your private repositories. This process ensures secure and seamless deployment of the Harness Self-Managed Enterprise Edition in restricted, offline environments.
```

File: docs/self-managed-enterprise-edition/self-managed-helm-based-install/install-in-an-air-gapped-environment.md [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+
+Air-gapped environments are characterized by a lack of direct access to the internet, which provides an added layer of security for sensitive data and systems. This isolation poses unique challenges to deploy and update software applications, as standard methods of accessing resources, such as Docker images, are not possbile.
```
Comment (douglas-j-bothwell) [inline docs/self-managed-enterprise-edition/self-managed-helm-based-install/install-in-an-air-gapped-environment.md:9]:
```suggestion
Air-gapped environments are characterized by a lack of direct access to the internet, which provides an added layer of security for sensitive data and systems. This isolation poses unique challenges to deploy and update software applications, as standard methods of accessing resources, such as Docker images, are not possible.
```

File: docs/self-managed-enterprise-edition/self-managed-helm-based-install/install-in-an-air-gapped-environment.md [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+
+- Harness Docker images [harness-docker-images.txt](https://storage.googleapis.com/smp-airgap-bundles/harness-docker-images.txt)
+- Harness air-gapped bundle [harness-airgapped.tgz](https://storage.googleapis.com/smp-airgap-bundles/harness-airgapped.tgz)
+- Harness airgap images [harness-airgap-images.sh](https://storage.googleapis.com/smp-airgap-bundles/harness-airgap-images.sh)
+
+## Save Docker images to your private registry
+
+After you download the required files, you must save your Docker images to your private registry.
+
+**To save your Docker images**
```
Comment (douglas-j-bothwell) [inline docs/self-managed-enterprise-edition/self-managed-helm-based-install/install-in-an-air-gapped-environment.md:50]: Note sure you need this paragraph, just repeats the heading

**Review by douglas-j-bothwell — DISMISSED** [review]

**Review by douglas-j-bothwell — APPROVED** [review]

**Pull request merged**